packages/tools/devtools/devtools-view/src/test/screenshot-tests/ScreenshotTestUtilities.tsx

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { FluentProvider } from "@fluentui/react-components";
+import React from "react";
+
+import { TelemetryNullLogger } from "@fluidframework/telemetry-utils";
+
+import { MockMessageRelay } from "../MockMessageRelay";
+import { MessageRelayContext } from "../../MessageRelayContext";
+import { LoggerContext } from "../../TelemetryUtils";
+import { getFluentUIThemeToUse } from "../../ThemeHelper";
+
+/**
+ * {@link TestContexts} input props.
+ */
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type TestContextsProps = React.PropsWithChildren<{}>;
+
+/**
+ * Wraps the input children in the contexts required by Devtools view components.
+ *
+ * 1. {@link MessageRelayContext}
+ *
+ * 2. {@link LoggerContext}
+ *
+ * 3. The required FluentUI theming context
+ */
+export function TestContexts(props: TestContextsProps): React.ReactElement {
```

Comment:
Seems like it doesn't need to be exported (same for the props).

Reply:
Ah, you're right. Not needed anymore. Will update.